lib/linguist/languages.yml

```diff
@@ -5088,6 +5080,24 @@ Standard ML:
   codemirror_mode: mllike
   codemirror_mime_type: text/x-ocaml
   language_id: 357
+Starlark:
+  type: programming
+  tm_scope: source.python
+  ace_mode: python
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
+  color: "#76d275"
+  extensions:
+  - ".bzl"
+  filenames:
+  - BUCK
+  - BUILD
+  - BUILD.bazel
+  - WORKSPACE
+  aliases:
+  - bazel
+  - bzl
```

Comment:
FWIW, `star` is another extension for Starlark files, especially when used outside of Bazel.

Reply:
@c4milo Please see https://github.com/github/linguist/pull/4759#discussion_r362961419.